File: app/src/main/java/piotr/michalkiewicz/mealplannerclient/user/model/NutritionProfileSettings.java
package piotr.michalkiewicz.mealplannerclient.user.model;

import java.io.Serializable;

import lombok.Data;
import lombok.NoArgsConstructor;
import piotr.michalkiewicz.mealplannerclient.user.model.enums.ActivityLevel;
import piotr.michalkiewicz.mealplannerclient.user.model.enums.Sex;

@Data
@NoArgsConstructor
public class NutritionProfileSettings implements Serializable {

    private Sex sex;
    private int height;
    private int weight;
    private int age;
    private int goal;
    private ActivityLevel activityLevel;
}

File: app/src/main/java/piotr/michalkiewicz/mealplannerclient/user/model/NutritionProfileSettings.kt
package piotr.michalkiewicz.mealplannerclient.user.model

data class NutritionProfileSettings( // TODO Bez nulli, jesli ktos nie ustawi tych ustawien to caly obiekt ma byc nullem
    val height: Int,
    val weight: Int,
    val age: Int,
    val goal: WeightGoal,
    val activityLevel: String
) {
    constructor() : this(
        -1,
        -1,
        -1,
        WeightGoal(80f, false),
        "Sedentary"
    )
}


